statvar_imports/database_on_indian_economy/india_rbi_state_statistics/agriculture_pvmap.csv

```diff
@@ -0,0 +1,83 @@
+key,prop,value,,,,,,,,,,
+State/Union Territory,observationAbout,{Data},,,,,,,,,,
+State/Union Territory/Zone,observationAbout,{Data},,,,,,,,,,
+Thousand Hectares,unit,Hectare,#Multiply,1000,,,,,,,,
+NET IRRIGATED AREA,populationType,LandCover,measuredProperty,area,landCoverType,IrrigatedLand,statType,netValue,,,name,"""Net irrigated area"""
```

Comment:
The statType as netValue and grossValue is not consistent with the existing variables. Left a comment in the CL with the reference.